recipes/_install.rb

```diff
@@ -29,16 +29,16 @@ def nsp
 include_recipe 'cerner_splunk::_restart_marker'
 
 # Actions
-# This service definition is used only for ensuring splunk is started during the run
-service 'splunk-start' do
+# This service definition is used for ensuring splunk is started during the run and to stop splunk service
+service 'splunk' do
   service_name service
   action :nothing
-  supports status: true, start: true
+  supports status: true, start: true, stop: true
   notifies :delete, 'file[splunk-marker]', :immediately
```

Comment:
I agree that it is a good idea to delete the marker. I was still thinking of a separate resource called splunk-stop (like I had before) so that we can avoid all the renaming.